File: Database/project_cpp/odbc_Add_Result.cpp
#include "odbc_db.h"
#include <iostream>
#include <string>
using namespace std;

int main(int argc, char *argv[])
{
string Username = "jrnolasc";            
string mysqlPassword = "aiK5usei";  
string SchemaName = "jrnolasc";          

   odbc_db myDB;
   myDB.Connect(Username, mysqlPassword, SchemaName);
   myDB.initDatabase();
 

   string builder = "";
   string query1 = "SELECT * from Result;";

   string GameId = "34";
   string TeamOneId = "NAME";
   string TeamTwoId = "popo";
   string ScoreOne = "ROASTING";
   string ScoreTwo = "popo";
   
   GameId = argv[1];
   TeamOneId= argv[2];
   TeamTwoId = argv[3];
   ScoreOne = argv[4];
   ScoreTwo = argv[5];
  
 
   
   string input = GameId + "," + TeamOneId + "," + TeamTwoId + "," + ScoreOne + ", " + ScoreTwo;               
   myDB.insert("Result", input);    
 
   builder.append("<br><br><br> Result" + myDB.quickquery(query1));
   cout << builder; 
        
   myDB.disConnect();//disconect Database

   return 0;
}

File: Database/project_cpp/odbc_Add_Result.php
<html>
<body>
<h3>Enter game result information you would like to add to the database:</h3>

<form action="odbc_Add_Result.php" method="post"> 
    Game ID: <input type="number"name="GameId"><br>
    Team One ID: <input type="number" name="TeamOneId"><br>
    Team Two ID: <input type="number" name="TeamTwoId"><br>
    Score One: <input type="number" name="ScoreOne"><br>
    Score Two: <input type="number" name="ScoreTwo"><br>
    <input name="submit" type="submit" >
</form>
<br><br>
<div style="position: absolute; bottom: 0; right: 0; width: 100px; text-align:right;">
    <a href=" http://www.csce.uark.edu/~jrnolasc/project_cpp/hello.html">HOME</a>
</div>

</body>
</html>

<?php
if (isset($_POST['submit'])) 
{
    // replace ' ' with '\ ' in the strings so they are treated as single command line args

    $GameId = escapeshellarg($_POST[GameId]);
    $TeamOneId = escapeshellarg($_POST[TeamOneId]);
    $TeamTwoId = escapeshellarg($_POST[TeamTwoId]);
    $ScoreOne = escapeshellarg($_POST[ScoreOne]);
    $ScoreTwo = escapeshellarg($_POST[ScoreTwo]);

    $command = '/home/jrnolasc/public_html/project_cpp/odbc_Add_Result.exe ' . $GameId .' '. $TeamOneId . ' ' . $TeamTwoId . ' ' . $ScoreOne . ' ' . $ScoreTwo;
    if ($command == "/home/jrnolasc/public_html/project_cpp/odbc_Add_Result.exe '' '' '' '' ''")
    {
        echo "No Game Found";
    }
    //echo '<p>' . 'command: ' . $command . '<p>';
    // remove dangerous characters from command to protect web server
    $command = escapeshellcmd($command);
 
    // run odbc_insert_item.exe
    system($command);           
}
?>